### DISCUSSION TITLE
Next minor version release

### DISCUSSION THREAD

**Discussion by rciummo-linkedin** [graphql-java/graphql-java#3228] [General]
Hi, 
My team is looking to bump to gql-java 20 but we need to leverage [Input Interceptor](https://github.com/graphql-java/graphql-java/pull/3188) to do so because the strict coercion updates are an issue for us.

Is it possible to release this soon in a minor version update?

Thanks,
Rosa

**Comment by dondonz** [comment]
Hi Rosa, in the current master we have reverted the stricter scalar coercion and added an input interceptor on top of that.

We recently formalised our [release policy](https://www.graphql-java.com/blog/release-policy). Expect the next version to be released in early July 2023.

If you urgently require the change (before July), you can alternatively use the latest master build version.

**Comment by dondonz** [comment]
Hey @rciummo-linkedin, @bbakerman made a good point that we ought to do a special release containing a revert of the stricter `parseValue` coercion to unblock you. The July release will include the input interceptor

20.3 should be landing in Maven in a few minutes https://github.com/graphql-java/graphql-java/releases/tag/v20.3

**Comment by rciummo-linkedin** [comment]
@dondonz @bbakerman Outside of the strict coercion, we actually do want to pull in the InputInterceptor for performance upgrades. We currently have a use case where we recreate the RawVariables map because we have to perform transformation on them and the InputInterceptor should provide improvements in that area that we want to test. Is there any way the input interceptor change can be released sooner than July?

**Reply by bbakerman** [reply]
We publish an artifact on every commit to master.

For example `0.0.0-2023-05-22T21-54-51-6b943f0` is a version that contains the commit `6b943f0`

So you can use those if you want to get to the latest code
